core/plugins/org.polarsys.capella.core.re/plugin.xml

```diff
@@ -454,11 +454,11 @@
             mapping="org.polarsys.capella.common.re"
             purpose="org.polarsys.capella.common.re.scopeElements">
          <scopeRetriever
-               class="org.polarsys.capella.core.re.handlers.scope.PartTypeRetriever"
+               class="org.polarsys.capella.core.transition.system.handlers.scope.PartTypeRetriever"
                id="org.polarsys.capella.core.re.handlers.scope.PartTypeRetriever">

```

Comment:
to rename id too

Reply:
AFAIU, no, the ID stays the same. The re plugin still is the declaring source of the retriever, it only reuses an implementation provided by transition.system.